x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/argument/Argument.java

```diff
@@ -144,6 +186,8 @@ static Argument fromParameter(javax.lang.model.util.Types types, VariableElement
      */
     void read(MethodSpec.Builder builder, boolean blockStyle);
 
+    void read(MethodSpec.Builder builder, String accessor, String firstParam);
```

Comment:
Probably needs javadoc like the adjacent methods.

Reply:
Also, since all but one implementation is an empty implementation, we could use a `default` implementation here in the interface?